tests/parm/regional_stoch.nml.IN

```diff
@@ -0,0 +1,330 @@
+    iopt_run = 1
+    iopt_sfc = 1
+    iopt_snf = 4
+    iopt_stc = 1
+    iopt_tbot = 2
+    iovr = 3
+    isol = 2
+    isot = 1
+    isubc_lw = 2
+    isubc_sw = 2
+    ivegsrc = 1
+    kice = 9
+    ldiag3d = .false.
+    lheatstrg = @[LHEATSTRG]
+    lndp_each_step = .true.
+    lndp_type = 0
+    lradar = .true.
+    lsm = 3
+    lsoil = 9
+    lsoil_lsm = 9
+    ltaerosol = .true.
+    lwhtr = .true.
+    n_var_lndp = 0
+    n_var_spp = @[N_VAR_SPP]
+    nsradar_reset = 3600
+    nst_anl = .true.
+    nstf_name = 2, 1, 0, 0, 0
+    oz_phys = @[OZ_PHYS_OLD]
+    oz_phys_2015 = @[OZ_PHYS_NEW]
+    pdfcld = .false.
+    pre_rad = .false.
+    print_diff_pgr = .false.
+    prslrd0 = 0.0
+    random_clds = .false.
+    redrag = .true.
+    satmedmf = @[SATMEDMF]
+    shal_cnv = .false.
+    swhtr = .true.
+    trans_trac = .true.
+    ttendlim = -999
+    use_ufo = .true.
+/
+
+&interpolator_nml
+    interp_method = 'conserve_great_circle'
+/
+
+&nam_sfcperts
+    iseed_lndp = 2010
+    lndp_lscale = 500000
+    lndp_prt_list = 0.2, 0.1
+    lndp_tau = 21600
+    lndp_type = 0
+    lndp_var_list = 'smc', 'vgf'
+/
+
+&nam_sppperts
+    iseed_spp = @[ISEED_SPP]
+    spp_lscale = @[SPP_LSCALE]
+    spp_prt_list = @[SPP_PRT_LIST]
+    spp_sigtop1 = @[SPP_SIGTOP1]
+    spp_sigtop2 = @[SPP_SIGTOP2]
+    spp_stddev_cutoff = @[SPP_STDDEV_CUTOFF]
+    spp_tau = @[SPP_TAU]
+    spp_var_list = @[SPP_VAR_LIST]
+/
+
+&nam_stochy
+    iseed_shum = 2
+    iseed_skeb = 3
+    iseed_sppt = 1
+    new_lscale = .true.
+    shum = 0.006
+    shum_lscale = 150000
+    shum_tau = 21600
+    shumint = 3600
+    skeb = 0.5
+    skeb_lscale = 150000
+    skeb_tau = 21600
+    skeb_vdof = 10
+    skebint = 3600
+    skebnorm = 1
+    sppt = 0.7
+    sppt_logit = .true.
+    sppt_lscale = 150000
+    sppt_sfclimit = .true.
+    sppt_tau = 21600
+    spptint = 3600
+    use_zmtnblck = .false.
+/
+
+&namsfc
+    fabsl = 99999
+    faisl = 99999
+    faiss = 99999
+    fnabsc = '../fix_lam/C3342.maximum_snow_albedo.tileX.nc'
+    fnacna = ''
+    fnaisc = '../fix_am/CFSR.SEAICE.1982.2012.monthly.clim.grb'
+    fnalbc = '../fix_lam/C3342.snowfree_albedo.tileX.nc'
+    fnalbc2 = '../fix_lam/C3342.facsf.tileX.nc'
+    fnglac = '../fix_am/global_glacier.2x2.grb'
+    fnmskh = '../fix_am/seaice_newland.grb'
+    fnmxic = '../fix_am/global_maxice.2x2.grb'
+    fnslpc = '../fix_lam/C3342.slope_type.tileX.nc'
+    fnsmcc = '../fix_am/global_soilmgldas.t126.384.190.grb'
+    fnsnoa = ''
+    fnsnoc = '../fix_am/global_snoclim.1.875.grb'
+    fnsotc = '../fix_lam/C3342.soil_type.tileX.nc'
+    fntg3c = '../fix_lam/C3342.substrate_temperature.tileX.nc'
+    fntsfa = ''
+    fntsfc = '../fix_am/RTGSST.1982.2012.monthly.clim.grb'
+    fnvegc = '../fix_lam/C3342.vegetation_greenness.tileX.nc'
+    fnvetc = '../fix_lam/C3342.vegetation_type.tileX.nc'
+    fnvmnc = '../fix_lam/C3342.vegetation_greenness.tileX.nc'
+    fnvmxc = '../fix_lam/C3342.vegetation_greenness.tileX.nc'
```

Comment:
All these static files must be located in te run directory, not in `../fix_am/` or `../fix_lam`. See other namelists for example.

Reply:
This is now fixed in the latest commit.